axiom/tests/test_span_event.c

```diff
@@ -256,6 +256,47 @@ static void test_span_event_category(void) {
   tlib_pass_if_str_equal("Category should be the one we set - http", "http",
                          nr_span_event_get_category(event));
 
+  nr_span_event_set_category(event, NR_SPAN_MESSAGE);
+  tlib_pass_if_str_equal("Category should be the one we set - message",
+                         "message", nr_span_event_get_category(event));
+
+  nr_span_event_destroy(&event);
+}
+
+static void test_span_event_spankind(void) {
+  nr_span_event_t* event = nr_span_event_create();
+
+  // Test : the default is NULL (spankind must be explicitly set)
+  tlib_pass_if_str_equal("The default category", NULL,
```

Comment:
```suggestion
  tlib_pass_if_str_equal("empty kind is NULL", NULL,
```

Reply:
Please see: https://github.com/newrelic/newrelic-php-agent/pull/991/commits/862604b081029206c2072275588076691d30560a

Reply:
https://github.com/newrelic/newrelic-php-agent/commit/862604b081029206c2072275588076691d30560a update is still not precise enough for my taste. But better than before.

Reply:
The initial suggestion of "empty" was imprecise, what does empty mean, how do you empty it?
The more precise wording would be to say that the default, in lieu of explicitly setting it, is NULL.
https://github.com/newrelic/newrelic-php-agent/pull/991/commits/4f91a63f1a1220ef144733f0abc4ed0dad9328c4